pacsman/integration_test.py

```diff
@@ -163,6 +164,29 @@ def test_local_studies_for_patient(local_client):
         assert ds.StudyInstanceUID
 
 
+@pytest.mark.integration
+@pytest.mark.local
```

Comment:
Any reason we are using `local` here instead of `remote`?

It's a bit confusing, but the way this is currently set up, `local` tests are not executed in CI. Furthermore, tests using the `remote_client` fixture (instead of the `local_client` fixture) do still test the filesystem dicom client in addition to the pynetdicom client. Thus, unless there's a reason why we cannot run this test in CI, I would prefer us to use the remote client here and change the pytest mark to `@pytest.mark.remote`.

I think that, generally, this integration test distinction between `local` and `remote` is a holdover from when we had a local Horos server and a remote third party PACS as a part of testing. At this point, Orthanc hosted in a docker container on CI should supercede both those use cases, so we probably can do away with `remote` and `local` entirely. However, that is outside the scope of this PR so no need to make those changes now.

Reply:
I think I just got in the coding flow and was going off tests around it. Updated to use `remote` in 03dfffa8c5d7fc16cf53f24ffecf550e59f7684b.

> It's a bit confusing, but the way this is currently set up [...]

I'm glad I'm not the only one that feels this way - definitely took me a few reads of this file on the last PR to understand what was going on 😅 . And definitely agree about fixing up being desirable future work - in fact, I think there are some tests that are actually broken, but never run. Seems like #42, #24, and #22 are all related and could be closed by one PR.